### ISSUE TITLE
[Feature Request] Option to Ignore Own Watchlist

### ISSUE THREAD

**Issue by AlliAlliAlli** [nylonee/watchlistarr#179]
I know this sounds a little odd, but if possible I'd love the ability to ignore my own watchlist and only sync my friends' watchlists. The reason being that I have a two separate setups of the Arrs, one for my users where everything is 1080p and another for myself which is all 4K. The info I put into Watchlistarr's config is for the 1080p Sonarr, Radarr, and Overseerr apps that I use for friends. The problem with that is if I add anything to my own watchlist, it ends up being added to those Arr instances instead of my personal 4K ones. So the ability to ignore my own watchlist would help a lot, that way I could either set up another instance of Watchlistarr only for myself or just use Overseerr's built-in function.

I'm not sure how simple or difficult something like this would be to implement so either way, thank you so much for all the hard work you've put into this app already. It's very helpful and I really appreciate being able to use it. :)

**Comment by Kpalsm** [comment]
Not sure if it'll work, but have you tried adding a second instance of watchlistarr for your separate arr stack, with your own settings? You'd have to use the same Plex API but point it at your other arr stack

**Comment by AlliAlliAlli** [comment]
@Kpalsm Without the option to ignore my own watchlist on the first Watchlistarr instance, I believe I'd just end up with my watchlist items being requested twice (once on my 1080p stack and once on my 4K stack).

**Comment by Kpalsm** [comment]
Yeah I think you're right about that. Hopefully one of the devs can be more helpful :)

**Comment by Jusremo** [comment]
Potential work-around would be making your own personal user account to watch with, add it to your Plex Home for easy access and Plex Pass sharing, set its watchlist to private.

I personally like to have my server management and personal account separate, solved a couple problems by having my setup that way.
